src/services/__tests__/phoenixAPI.js

```diff
@@ -70,3 +70,39 @@ describe("GET status of the game", () => {
     );
   });
 });
+
+describe("GET winning indices of the game", () => {
+
+  it("fetches any winning indices for a board", async () => {
+    // setup
+    fakeAxios.get.mockImplementation(() =>
+      Promise.resolve({
+        data: { index_list: ["1", "2", "3"] }
+      })
+    );
+
+    // work
+    const data = await PhoenixApi.requestWinningIndices(["X", "X", "X", "O", "O", "6", "7", "8", "9"]);
+
+    // expect
+    expect(data.index_list).toEqual(["1", "2", "3"]);
```

Comment:
if you're going to be expecting the same set of data can we extract it to a variable?
ex:

`const winningIndices = ["1", "2", "3"];`

```
      Promise.resolve({
        data: { index_list: winningIndices }
      })
```

`expect(data.index_list).toEqual(winningIndices);`